pytest/lib/cluster.py

```diff
@@ -412,6 +412,16 @@ def start_proxy_if_needed(self):
         if self._start_proxy is not None:
             self._proxy_local_stopped = self._start_proxy()
 
+    def output_logs(self):
+        if os.environ.get('BUILDKITE'):
+            logger.info('=== stdout: ')
+            logger.info(self.stdout_name.read_text('utf-8', 'replace'))
+            logger.info('=== stderr: ')
+            logger.info(self.stderr_name.read_text('utf-8', 'replace'))
+        else:
+            logger.info(f'=== stdout: available at {self.stdout_name}')
+            logger.info(f'=== stderr: available at {self.stderr_name}')
```

Comment:
Those two lines aren’t true. When nodes run they use `test#` as their home directory but once they are finished those directories are renamed to `test#_finished`. Those two lines will print the old path.

Reply:
Using `self.node_dir` doesn’t really change the output. `self.node_dir` points at the `test#` directory.

Reply:
I made sure to set `self.node_dir` to the value of the moved directory and and also moved calls to `output_logs` to after the files are moved to their final destination.

Reply:
(In retrospective I should've made more commits to better demonstrate changes being made…)

Reply:
Sorry, missed that change.